src/Package/Test/DataInspect/ViewersTest.cs

```diff
@@ -18,19 +18,23 @@
 using Microsoft.VisualStudio.R.Package.Shell;
 using NSubstitute;
 using Xunit;
+using Microsoft.UnitTests.Core.XUnit.MethodFixtures;
 
 namespace Microsoft.VisualStudio.R.Package.Test.DataInspect {
     [ExcludeFromCodeCoverage]
     [Collection(CollectionNames.NonParallel)]   // required for tests using R Host 
     [Category.Viewers]
     public class ViewersTest : IAsyncLifetime {
+        private const REvaluationResultProperties AllFields = unchecked((REvaluationResultProperties)~0);
+
+        private readonly TestMethodFixture _testMethod;
         private readonly IRSessionProvider _sessionProvider;
         private readonly IObjectDetailsViewerAggregator _aggregator;
         private readonly IRInteractiveWorkflow _workflow;
 
-        public ViewersTest() {
+        public ViewersTest(TestMethodFixture testMethod) {
+            _testMethod = testMethod;
```

Comment:
I think it's not actually needed, this was from an earlier iteration when I was trying to start the session differently. But it's a port of the other commit that just went in, so I'd rather not touch anything here unless it's specifically required to merge into this branch.